keel-orca/src/main/kotlin/com/netflix/spinnaker/keel/model/OrcaNotification.kt

```diff
@@ -60,6 +60,7 @@ data class NotificationMessage(
 const val RAINBOW = "\uD83C\uDF08"
 const val THUNDERCLOUD = "\u26c8\ufe0f"
 
+//TODO [gyardeni]: figure out if we want to keep it around, or deprecate it
```

Comment:
I think you need to remove the actual call to this function here to avoid duplicate notifications from Orca and Keel:
https://github.com/spinnaker/keel/blob/9db12ff2949db3c38501747e6cae3de3ec11fc6f/keel-orca/src/main/kotlin/com/netflix/spinnaker/keel/orca/OrcaTaskLauncher.kt#L97

Reply:
I'm going to address it in my next PR.